spec/features/admins_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
```

Comment:
this package looks like it could potentially be vulnerable and left unmaintained 
<img width="938" alt="screen shot 2018-09-05 at 8 28 42 pm" src="https://user-images.githubusercontent.com/32852965/45129607-58c80580-b14a-11e8-87df-089e348bd6d7.png">

Reply:
rails_helper isn't a gem - this is just requiring the rails_helper.rb file located in the spec folder that contains all our tests